robot/EDA/tests/browser/settings/system.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/SystemSettingsPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Capture screenshot and delete records and close browser
+
+*** Test Cases ***
+Verify standard field values
+    [Documentation]         Checks for the default account model value as Administrative and checks store errors
+    ...                     checkbox is checked and send error notification checkbox is unchecked and error
+    ...                     notification recipient dropdown is set to All Sys Admins and checkboxes for disable
+    ...                     error handling and automatic household naming are unchecked and administrative 
+    ...                     account name format is set to {!LastName} Administrative and household account
+    ...                     name format dropdown is set to {!LastName} Household.
```

Comment:
You don't have explicitly mention each value in documentation, since it is clearly readable in the script itself.
Maybe something along the lines like:

> Verifies the default values for all checkboxes and dropdowns in System tab of EDA Settings

Reply:
Noted..thanks for the suggestion